meta-balena-common/recipes-connectivity/bluez5/bluez5/run-ptest

```diff
@@ -0,0 +1,31 @@
+#! /bin/sh
```

Comment:
We remove the `ptest` feature from the balena-os distro settings, so all ptest related code can be removed.

Reply:
Removed in a separate commit.